sidekick/lib/src/init/init_command.dart

```diff
@@ -22,8 +22,7 @@ class InitCommand extends Command {
     argParser.addOption(
       'cliName',
       abbr: 'n',
-      help:
-          'The name of the CLI to be created \nThe `_cli` prefix, will be define automatically',
+      help: 'The name of the CLI to be created \nThe `_cli` prefix, will be defined automatically',
```

Comment:
Change your formate to LineLength of 80

Reply:
Oh nooo I hate it

Reply:
But why doesn't the CI catch it, tho? 🤔

Reply:
Good catch [b16a5be](https://github.com/phntmxyz/sidekick/pull/21/commits/b16a5be2e01bb270c69ad871605c2cff1ae1c2f0)